instrumentation/google.golang.org/grpc/otelgrpc/interceptor_test.go

```diff
@@ -372,6 +372,59 @@ func TestStreamClientInterceptor(t *testing.T) {
 	_ = streamClient.CloseSend()
 }
 
+// TestStreamClientInterceptorCancelContext tests a cancel context situation.
+// There should be no goleaks
+func TestStreamClientInterceptorCancelContext(t *testing.T) {
```

Comment:
Cannot we simply modify the `TestStreamClientInterceptor` instead of creating a new test?

Reply:
I'm not sure.. because, as I understand, the `TestStreamClientInterceptor` demonstrates normal stream termination receiving `receiveEndEvent`. `TestStreamClientInterceptorCancelContext` ends the stream by closing the context.